### ISSUE TITLE
GC 3.4 Crash when changing target power during a custom erg mode workout

### ISSUE THREAD

**Issue by devenvag** [GoldenCheetah/GoldenCheetah#2375]
Devices ->
- HR : Garmin HRM3-SS
- Power & cadence : PowerTap p1
- Trainer : Tacx Smart Bushido

Plateform & GC ->
- macOS Sierra 10.12.3
- GC 3.4 Build id : 3957

To reproduce ->
1/ Choose a a erg mode workout
2/ Start the workout
3/ Change target power
4/ Crash

**Comment by liversedge** [comment]
How are you changing target power?

**Comment by devenvag** [comment]
By clicking on up and arrow or on the vertical cursor, the result is the same.

**Comment by liversedge** [comment]
I don't follow -- is this in the workout editor ?

**Comment by devenvag** [comment]
Yes in the live workout view when you're doing an erg mode workout and changing the target power with the buttons that I highlighted below in a red rectangle. It is not on every click but for me 3 or 4 clicks and GC crashes.
![button_crash](https://cloud.githubusercontent.com/assets/9590423/23581332/1d20518e-0112-11e7-91f9-4b4c94088604.png)

I hope it is more clear.

**Comment by liversedge** [comment]
Ah thanks. Can you see if the dev build fixes it, since I cannot recreate ?

**Comment by devenvag** [comment]
Just tested and it seems to work with the last 3.5 dev build. I will do
better tests tomorow with a long workout.

2017-03-04 20:02 GMT+01:00 Mark Liversedge <notifications@github.com>:
